lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -281,6 +285,14 @@ module InternalConstants
     # Cloud Logging API.
     config_param :use_grpc, :bool, :default => false
 
+    # Whether valid entries should be written even if some other entries fail
+    # due to INVALID_ARGUMENT or PERMISSION_DENIED errors when communicating to
+    # the Cloud Logging API. This is highly recommended.
+    # TODO(qingling128): Remove the comment below when the support is added for
```

Comment:
This TODO would appear in the generated docs. I would instead mention this in the TODO on the warning.

Reply:
Changed.